pkg/utils/openshift/utils.go

```diff
@@ -1,45 +1,24 @@
 package openshift
```

Comment:
@bmozaffa 

- all platform code moved to new 'platform' pkg
- utils.go simplified - 2 funcs w/usage comments to better present simple & user-friendly tool